src/server/rpc/procedures/azure-translation/azure-translation.js

```diff
@@ -20,6 +20,19 @@ TranslationConsumer._get_guid = function () {
     });
 };
 
+/**
+ * Customized cache key to allow translated strings to be cached even though the URL may be identical
+ * @param {Object} queryOptions 
+ */
+TranslationConsumer._getCacheKey = function(queryOptions){
+    let parameters = [];
+    parameters.push(queryOptions.method || 'GET');
+    let fullUrl = (queryOptions.baseUrl || this._baseUrl) + queryOptions.queryString;
+    parameters.push(fullUrl);
+    if (queryOptions.body) parameters.push(JSON.stringify(queryOptions.body));
```

Comment:
is this the only line that's changed? 
This addresses a good issue with this method but it's not specific to this pull request and other services could use it too. I think we could pull it out and apply it to `ApiConsumer` directly.

Reply:
Yeah, it needed that because of how the body was structured. I can pull it out and make it a PR for the `ApiConsumer` instead.

Reply:
sounds good to me

Reply:
It's taken out now.